app/src/main/java/com/example/atlaskotlindemo/ui/dashboard/DashboardFragment.kt

```diff
@@ -63,14 +64,14 @@ class DashboardFragment : Fragment() {
                     return@launch
                 }
 
-                (requireActivity().application as AtlasDemoApplication).atlasSdk.identify(userId = id)
+                atlasSdk.identify(userId = id)
                 Log.d("Atlas", "Logged in as $id")
             }
         }
 
         binding.logoutButton.setOnClickListener {
             viewLifecycleOwner.lifecycleScope.launch {
-                (requireActivity().application as AtlasDemoApplication).atlasSdk.identify()
+                atlasSdk.identify()
```

Comment:
You have replaced the `suspend` with `CompletableFuture.runAsync`. I wonder if there is anything should be used in the way you use it outside?